extensions/hibernate-orm/runtime/pom.xml

```diff
@@ -75,6 +75,12 @@
                     <groupId>javax.activation</groupId>
                     <artifactId>javax.activation-api</artifactId>
                 </exclusion>
+
+                <!-- Not necessary at runtime -->
+                <exclusion>
+                    <groupId>org.jboss</groupId>
+                    <artifactId>jandex</artifactId>
+                </exclusion>
```

Comment:
I'm not sure I understand this PR given its title. This was an exclusion so it was never necessary at runtime.

Do you mean that hibernate-core doesn't depend on jandex anymore?

Reply:
This is a new exclusion - I'm not removing an exclusion?

`hibernate-core` still has the dependency, as it normally makes use of it, but you won't need it in Quarkus as things are processed at deployment time

Reply:
Hu. Sorry, I don't know what happened, I somehow read it backwards. That's very odd :).

Sorry about the noise.